.github/workflows/build-artifacts.yml

```diff
@@ -29,20 +29,20 @@ jobs:
           - platform: windows-x86_64
             os: windows-2022
             triplet: x64-windows-release
-          # We can safely use the latest image, since the real job will run in the manylinux container.
+          # Use manylinux image before Python3.9 support was dropped on 10/01/25.
           - platform: linux-x86_64
             os: ubuntu-latest
-            manylinux: quay.io/pypa/manylinux_2_28_x86_64
+            manylinux: quay.io/pypa/manylinux_2_28_x86_64<2025.10.01-1
```

Comment:
How about using `=` instead since this is a "pin" ? And a comment explaining why we are pining it so that we don't forget to remove this restriction when python 3.9 support requirement is dropped ?